File: FitnessWeb.API/Commands/CreateFitnessExerciseCommand.cs
using MediatR;

namespace FitnessWeb.API.Commands
{
    public class CreateFitnessExerciseCommand : IRequest<int>
    {
        public int FitnessTrainingId { get; set; }
        public string Name { get; set; }
        public int CountOfRepeats { get; set; }
        public string VideoUrl { get; set; }
    }
}

File: FitnessWeb.Frontend/AngularApp/src/commands/createFitnessExerciseCommand.ts
export class CreateFitnessExerciseCommand {
    name: string;
    countOfRepeats: number;
    videoUrl: string;
    fitnessTrainingId: number;

    constructor(name: string, countOfRepeats: number, videoUrl: string, fitnessTrainingId: number) {
        this.name = name;
        this.countOfRepeats = countOfRepeats;
        this.videoUrl = videoUrl;
        this.fitnessTrainingId = fitnessTrainingId;
    }
}
